components/nav/TopLevelMenu.vue

```diff
@@ -64,7 +64,7 @@ export default {
           label:   x.nameDisplay,
           ready:   x.isReady,
           osLogo:  x.providerOsLogo,
-          logo:    x.providerLogo,
+          logo:    x?.status?.provider === 'harvester' ? require(`~/assets/images/providers/kubernetes.svg`) : x.providerLogo,
```

Comment:
Only the logo in the menu needs to be overridden, so I don't think it can be handled inside the model.

We think we should not use the harvester logo in the EXPLORE CLUSTER, because what the user sees here is the container-related information
![image](https://user-images.githubusercontent.com/24985926/133189955-8f1611ae-7c9b-4be2-8db9-00b7dc08d191.png)

Reply:
I think this should still be pushed into the mgmt cluster model as a new property, something like `providerMenuLogo` (this fits other ways to determine the logo).

Reply:
Done